test/__snapshots__/test.js.snap

```diff
@@ -198,9 +201,37 @@ const iface = {
     }
     return obj;
   },
-  interface: DOMImplementation,
-  expose: {
-    Window: { DOMImplementation }
+
+  install(globalObject) {
+    const ctor = function DOMImplementation() {
+      throw new TypeError(\\"Illegal constructor\\");
+    };
+
+    const proto = {};
+
+    Object.defineProperties(proto, {
+      ...Object.getOwnPropertyDescriptors(DOMImplementation.prototype),
+      constructor: {
+        writable: true,
+        configurable: true,
+        value: ctor
+      }
+    });
+    Object.defineProperty(ctor, \\"prototype\\", {
+      writable: false,
+      value: proto
+    });
+
+    if (globalObject[ctorRegistry] === undefined) {
+      globalObject[ctorRegistry] = {};
```

Comment:
Maybe `Object.create(null)` would be better, to avoid treating e.g. `hasOwnProperty` as a constructor.